stringcount.c

```diff
@@ -0,0 +1,24 @@
+//Bettina Benitez
+#include <stdio.h>
+#include <string.h>
```

Comment:
Hmm, what were you trying to do in this program? Also, did you hand in code for the hangman problem?